### PR TITLE
[Do not merge]Add appnav tests

### PR DESCRIPTION
**Pull request by sivareddypv1**
locally updated page objects used for the test. We will merge the test once the PO updates published to salesforce PO artifact.

### CODE REVIEW

File: utam-tests/src/test/java/utam/examples/salesforce/web/AppNavigationTests.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    setupChrome();
+    login(testEnvironment, "home");
+  }
+
+  /**
+   * navigate to object home via URL and click New
+   *
+   * @param recordType record type affects navigation url
+   */
+  private void openRecordModal(RecordType recordType) {
```
Comment (lizaiv77) [inline utam-tests/src/test/java/utam/examples/salesforce/web/AppNavigationTests.java:45]: why this method is here? pls remove

File: utam-tests/src/test/java/utam/examples/salesforce/web/AppNavigationTests.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  @Test
+  public void testNavigateToNanBarOverflowItem() {
+    getDriver().get(testEnvironment.getRedirectUrl());
+    log("Load Desktop layout container");
+    DesktopLayoutContainer layoutContainer = from(DesktopLayoutContainer.class);
+
+    AppNavBar navBar= layoutContainer.getAppNav().getAppNavBar();
+    log("Navigate to overflow menu item");
+    navBar.getShowMoreMenuButton().expand();
+//    menu item with name 'Forecasts' should present in overflow items
```
Comment (lizaiv77) [inline utam-tests/src/test/java/utam/examples/salesforce/web/AppNavigationTests.java:82]: code format pls

File: utam-preview/src/main/resources/compiler.config.json [hunk 3]
```diff
@@ -29,6 +29,10 @@
     {
       "typeMatch": "utam-portal",
       "pathMatch": ".*/portal"
+    },
+    {
+      "typeMatch": "utam-nav",
+      "pathMatch": ".*/nav"
```
Comment (lizaiv77) [inline utam-preview/src/main/resources/compiler.config.json:35]: pls remove and replace all page objects by artifact

**Review by lizaiv77 — CHANGES_REQUESTED** [review]
pls wait for new version 1.1.1 on https://mvnrepository.com/artifact/com.salesforce.utam/salesforce-pageobjects and address comments

File: utam-tests/src/test/java/utam/examples/salesforce/web/AppNavigationTests.java (hunk 2 above)
Comment (sivareddypv1) [reply utam-tests/src/test/java/utam/examples/salesforce/web/AppNavigationTests.java:82]: formatted

**Pull request merged**